alvr/dashboard/js/app/monitor.js

```diff
@@ -585,20 +586,12 @@ define([
             Array(length).fill(null),
             Array(length).fill(null),
             Array(length).fill(null),
-            Array(length).fill(null),
-            Array(length).fill(null),
-            Array(length).fill(null),
-            Array(length).fill(null),
-            Array(length).fill(null),
-            Array(length).fill(null),
-            Array(length).fill(null),
-            Array(length).fill(null),
```

Comment:
Can we have these lines not repeat? `...Array(7).fill(null).map(x => Array(length).fill(null))`